src/app/zap-templates/zcl/data-model/chip/joint-fabric-datastore-cluster.xml

```diff
@@ -0,0 +1,370 @@
+  </struct>
+
+  <enum name="DatastoreAccessControlEntryAuthModeEnum" type="enum8">
+    <cluster code="0x0752"/>
+    <item name="PASE" value="0x01"/>
+    <item name="CASE" value="0x02"/>
+    <item name="Group" value="0x03"/>
+  </enum>
+
+  <struct name="DatastoreAccessControlTargetStruct">
+    <cluster code="0x0752"/>
+    <item fieldId="0" name="Cluster" type="cluster_id" isNullable="true"/>
+    <item fieldId="1" name="Endpoint" type="endpoint_no" isNullable="true"/>
+    <item fieldId="2" name="DeviceType" type="devtype_id" isNullable="true"/>
+  </struct>
+
+  <struct name="DatastoreAccessControlEntryStruct">
+    <cluster code="0x0752"/>
+    <item fieldId="1" name="Privilege" type="DatastoreAccessControlEntryPrivilegeEnum" isFabricSensitive="true" min="0x00" max="0x05"/>
+    <item fieldId="2" name="AuthMode" type="DatastoreAccessControlEntryAuthModeEnum" isFabricSensitive="true" min="0x00" max="0x03"/>
+    <item fieldId="3" name="Subjects" type="int64u" isNullable="true" array="true" isFabricSensitive="true"/>
+    <item fieldId="4" name="Targets" type="DatastoreAccessControlTargetStruct" isNullable="true" array="true" isFabricSensitive="true"/>
+  </struct>
+
+  <struct name="DatastoreACLEntryStruct" isFabricScoped="true">
+    <cluster code="0x0752"/>
+    <item fieldId="0" name="NodeID" type="node_id"/>
+    <item fieldId="1" name="ListID" type="int16u"/>
+    <item fieldId="2" name="ACLEntry" type="DatastoreAccessControlEntryStruct"/>
+    <item fieldId="3" name="StatusEntry" type="DatastoreStatusEntryStruct"/>
+  </struct>
+
+  <struct name="DatastoreNodeInformationEntryStruct" isFabricScoped="true">
+    <cluster code="0x0752"/>
+    <item fieldId="1" name="NodeID" type="node_id"/>
+    <item fieldId="2" name="FriendlyName" type="char_string" length="32"/>
+    <item fieldId="3" name="CommissioningStatusEntry" type="DatastoreStatusEntryStruct"/>
+  </struct>
+
+  <struct name="DatastoreAdministratorInformationEntryStruct" isFabricScoped="true">
+    <cluster code="0x0752"/>
+    <item fieldId="1" name="NodeID" type="node_id"/>
+    <item fieldId="2" name="FriendlyName" type="char_string" length="32"/>
+    <item fieldId="3" name="VendorID" type="vendor_id"/>
+    <item fieldId="4" name="ICAC" type="long_octet_string" length="400"/>
+  </struct>
+
+  <enum name="DatastoreGroupKeySecurityPolicyEnum" type="enum8">
+    <cluster code="0x0752"/>
+    <item name="TrustFirst" value="0x00"/>
+    <item name="CacheAndSync" value="0x01"/>
+  </enum>
+
+  <enum name="DatastoreGroupKeyMulticastPolicyEnum" type="enum8">
+    <cluster code="0x0752"/>
+    <item name="PerGroupID" value="0x00"/>
+    <item name="AllNodes" value="0x01"/>
+  </enum>
+
+  <struct name="DatastoreGroupKeySetStruct">
+    <cluster code="0x0752"/>
+    <item fieldId="0" name="GroupKeySetID" type="int16u"/>
+    <item fieldId="1" name="GroupKeySecurityPolicy" type="DatastoreGroupKeySecurityPolicyEnum" isFabricSensitive="true" min="0x00" max="0x01"/>
```

Comment:
Spec issue: all these isFabricSensitive bits are nonsense because this struct is not fabric-scoped.  Again, the codegen ends up just ignoring them.